### ISSUE TITLE
Import error

### ISSUE THREAD

**Issue by VincentAuguin** [pdollar/coco#35]
Hi,

I get an import error ```ImportError: pycocotools/_mask.so: undefined symbol: PyFPE_jbuf``` when i try to run first block of ```pyCocoDemo.ipynb```

I've already run ```make``` and no error occured.

Have you any lead ?

**Comment by tylin** [comment]
A possible solution from stackoverflow: http://stackoverflow.com/questions/36190757/numpy-undefined-symbol-pyfpe-jbuf

**Comment by facelessmanx** [comment]
I am getting this same error. It works just fine on my system but results in an error when i run it on a cloud  system.

**Comment by xch20010** [comment]
Have you solved this error?

**Comment by spnaik** [comment]
I have the same error. How did you solve it ?

**Comment by ClassicDaniel** [comment]
For me, using the following fork did the trick:

https://github.com/philferriere/cocoapi.git

**Comment by DonghunP** [comment]
Could someone give us a help? I have a same problem...

  import pycocotools._mask as _mask
ImportError: /home/anaconda/PycharmProject/Self-DrivingCar/models/research/object_detection/coco/PythonAPI/pycocotools/_mask.cpython-35m-x86_64-linux-gnu.so: undefined symbol: PyFPE_jbuf

**Comment by ouyangkid** [comment]
@ClassicDaniel Yes, the trick is mkdir a subdir to apart the ipynb files from the coco source. There is no need to change a fork.

**Comment by csufangyu** [comment]
> Could someone give us a help? I have a same problem...
> 
> import pycocotools._mask as _mask
> ImportError: /home/anaconda/PycharmProject/Self-DrivingCar/models/research/object_detection/coco/PythonAPI/pycocotools/_mask.cpython-35m-x86_64-linux-gnu.so: undefined symbol: PyFPE_jbuf
I meet a same problem.but if I use python3,it would not meet it,could you solve it in python2?

**Comment by wangliting-amy** [comment]
i meet the same problem when i want to ues cocoeval. so i just do "python setup.py build_ext --inplace" and "python setup.py build_ext install" when i open new terminal do same commands again.it works for me.

**Comment by ou525** [comment]
Run the setup.py file to solve this problem

**Comment by stevenji** [comment]
Maybe you  have multi python version， makesure you use the same version of python, and delete the .so  in dir, then run the setup.py again.

**Comment by wangguanbei** [comment]
1.pip uninstall pycocotools
2.in cocoapi's git folder:
pip install git+https://github.com/philferriere/cocoapi.git#subdirectory=PythonAPI
